package/yast2-services-manager.spec

```diff
@@ -33,7 +33,7 @@ Source0:        %{name}-%{version}.tar.bz2
 
 Requires:       ruby
 # ServicesManager library
-Requires:       yast2 >= 3.1.86
+Requires:       yast2 >= 3.2.37.2
```

Comment:
Why is this requirement different from the `BuildRequires`?

Reply:
overlook, thanks for notice